.github/workflows/gcloud-sdk-test.yml

```diff
@@ -0,0 +1,19 @@
+name: Reusable google cloud sdk setup test
+
+on:
+  pull_request:
+    branches: [ main ]  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2   
+    - name: Install dependencies
+      uses: 'hagaltech/common-workflows/setup-gcloud-sdk@add-gcloud-action'
+      with:
+        gcp_credentials: ${{ secrets.GCP_SA_KEY }}
+      
+    - run: |
+        gcloud info
```

Comment:
Should we expose path to our registry here as this is public repo? May be we can consider to it is SECRET?

Reply:
Made the registry name a secret :+1: